### DISCUSSION TITLE
Huge barrier to go from idea to proposal?

### DISCUSSION THREAD

**Discussion by 1player** [cooklang/spec#88] [General]
There's a lot of interesting proposals here in the discussions, some of which are very simple, but to go from discussion to a proposal on needs to fill a very bureaucratic proposal template asking motivations, detailed description, reasoning, impact on present and future implementations etc. I get why projects like Rust have a similar set up, but is Cooklang that big and complex to require all that ceremony?

For a simple language in its infancy it's very discouraging to go through all that rigmarole, and in fact no one has even submitted a proposal yet. Having a simpler template and most of the discussion being done on the PR itself would be preferable in my opinion.

I started this discussion because I wanted to submit a pull request for the optional ingredient syntax discussed at https://github.com/cooklang/spec/discussions/50 but I have to admit I didn't expect to have to spend an hour to explicitly codify all single aspects and motivations behind that feature.

**Comment by dubadub** [comment]
Hi @1player, thanks for bringing it up. Yes, I think you're right. I'll make it easier, at least many things can be optional.
